data.sql

```diff
@@ -1,5 +1,8 @@
 /* Populate database with sample data. */
 
-INSERT INTO animals (name) VALUES ('Luna');
-INSERT INTO animals (name) VALUES ('Daisy');
-INSERT INTO animals (name) VALUES ('Charlie');
+INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg)
+VALUES 
+("Agumon", "2020-02-03", 0, true, 10.23), 
+("Gabumon", "2018-11-15", 2, true, 8.0),
+("Pikachu", "2021-01-07", 1, false, 15.04),
+("Devimon", "2017-05-12", 5, false, 11.0);
```

Comment:
- [ ] **Required:** Kind update the neutered value of `Devimon` to true as required.

Reply:
I have updated the data and query no. 6 that is affected